### PR TITLE
Update Buildkite plugins in docs

### PR DESCRIPTION
**Pull request by samchungy**

### CODE REVIEW

File: .github/renovate.json5 [hunk 1]
```diff
@@ -112,5 +112,18 @@
...
       datasourceTemplate: 'npm',
     },
+    {
+      customType: 'regex',
+      fileMatch: ['.md$'],
+
+      // Matches Buildkite plugins but preserves versions in the before diffs
+      // - docker-compose#v2.0.0: PASS
+      // - - docker-compose#v2.0.0: FAIL
+      // + - docker-compose#v2.0.0: PASS
```
Comment (72636c) [inline .github/renovate.json5:122]: Next level shit right here

File: .github/renovate.json5 [hunk 2]
```diff
@@ -112,5 +112,18 @@
       ],
       datasourceTemplate: 'npm',
     },
+    {
+      customType: 'regex',
+      fileMatch: ['.md$'],
```
Comment (72636c) [inline .github/renovate.json5:117]: Just thinking out loud: would we want to ignore `CHANGELOG.md` entirely, particularly if we apply this to a Buildkite plugin repo in future?

File: .github/renovate.json5 [hunk 3]
```diff
@@ -112,5 +112,18 @@
...
+
+      // Matches Buildkite plugins but preserves versions in the before diffs
+      // - docker-compose#v2.0.0: PASS
+      // - - docker-compose#v2.0.0: FAIL
+      // + - docker-compose#v2.0.0: PASS
+      matchStrings: [
+        '^[s+]*- (?<depName>[a-zA-Z0-9-]+?)#v(?<currentValue>.*?):',
```
Comment (72636c) [inline .github/renovate.json5:124]: Should the `depName` pattern include `/` for e.g. `seek-oss/docker-ecr-cache`?

**Review by 72636c — APPROVED** [review]

File: .github/renovate.json5 (hunk 3 above)
Comment (samchungy) [reply .github/renovate.json5:124]: Oh yes.

**Pull request merged**